### ISSUE TITLE
Exporting in arabic (non-english) languages is not working

### ISSUE THREAD

**Issue by obaid** [Botmock/botmock-rasa-export#30]
**Issue Overview**
When exporting a project with non-english language (i.e. Arabic etc) the intent utterances are not exported properly

**Steps To Reproduce**
1. Create a project with arabic intents (utterances)
2. Export the project
3. See the intent file generated

**Expected Behavior**
It should have intents utterances

**Screenshots**
N/A

**Comment by alshell7** [comment]
Seems like the API is not emitting the utterances (training phrases).

Tried to check by hitting the API straight away, to fetch the intents:

*On NodeJS* -> `https` package:


```javascript
var http = require("https");

var options = {
  "method": "GET",
  "hostname": "app.botmock.com",
  "port": null,
  "path": "/api/teams/<TEAM_ID>/projects/<PROJECT_ID>/intents",
  "headers": {
    "content-type": "application/json",
    "authorization": "Bearer <API_TOKEN>"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();
```


And the results, from one of the Arabic botmock projects we have, were:

```js

{
  id: '78968570-bab7-11ea-8342-97bc16942767',
  name: 'city_coverage',
  utterances: [],
  created_at: {
    date: '2020-06-30 09:52:55.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  updated_at: {
    date: '2020-07-01 12:18:09.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  is_global: true,
  slots: [
    {
      id: 'a58dcbbf-3e07-418d-9536-929bbb946781',
      variable_id: '8dc448b0-bab7-11ea-b174-4f22dcbb72f6',
      is_required: false,
      prompt: 'الرجاء ذكر المدينة التي تسكن فيها '
    }
  ]
}
{
  id: 'bf5b6ec0-bb0c-11ea-ae3a-6d174f304b08',
  name: 'intent_yes',
  utterances: [],
  created_at: {
    date: '2020-06-30 20:03:21.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  updated_at: {
    date: '2020-06-30 20:03:21.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  is_global: false,
  slots: []
}
{
  id: '95724ae0-bb9f-11ea-911f-c748ec581533',
  name: 'missing_order',
  utterances: [],
  created_at: {
    date: '2020-07-01 13:34:26.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  updated_at: {
    date: '2020-07-01 13:34:26.000000',
    timezone_type: 3,
    timezone: 'UTC'
  },
  is_global: false,
  slots: []
}
```

**Comment by nonnontrivial** [comment]
Closing this, as it should be resolved as of a575592b962a47ac694af3d991b123f1a149b946 .
